lighthouse-core/gather/driver.js

```diff
@@ -276,8 +276,9 @@ class Driver {
    * @return {Promise<LH.CrdpCommands[C]['returnType']>}
    */
   sendCommand(method, ...params) {
-    const timeout = this._nextProtocolTimeout;
-    this._nextProtocolTimeout = DEFAULT_PROTOCOL_TIMEOUT;
+    const timeout = typeof this._nextProtocolTimeout === 'undefined' ?
+      DEFAULT_PROTOCOL_TIMEOUT : this._nextProtocolTimeout;
+    this._nextProtocolTimeout = undefined;
```

Comment:
hmm, I don't follow what this is doing. It seems the default state goes to `undefined`, which now gets interpreted as "use the default", but why was that needed for an extra signal?

Reply:
removed it all 👍